mods/ca/rules/gdi/commander-tree.yaml

```diff
@@ -70,7 +70,7 @@ commander_tree.Mammoth_Tank_GDI.Reflector:
 	ProvidesPrerequisite@filler:
 		Prerequisite: filler.gdimammoth2
 
-commander_tree.Mammoth_Tank_GDI.PDL:
```

Comment:
be careful with such changes as it has to be reflected everywhere (prerequisites)

also I have a commit where I have changed all PLD to PDL as Point Defense Laser should be the right term, I confused it at the beginning and called everything point laser defense, which was wrong

https://github.com/patrickwieth/YMCA/commit/2d8922c0f3801baddeee6d31ca3d79e48ba85ce6

Reply:
ill keep that in mind then